app/src/main/java/org/p4/p4d2/tutorial/Ipv6RoutingComponent.java

```diff
@@ -586,14 +591,12 @@ public void event(LinkEvent event) {
 
             if (mastershipService.isLocalMaster(srcDev)) {
                 executorService.submit(() -> {
-                    setUpMyStationTable(srcDev);
```

Comment:
By removing this the my_station table is not populated after reloading the app, isn't it? Is this expected?

Reply:
I think we can call it once device is up and available, see DeviceLIstener

Reply:
After an app reload (where we clean all flow rules), are you sure a device event will be triggered for each device?

Reply:
this case can be handled by `setUpAllDevices` method I think

Reply:
Oh right, sorry...  I mistakenly thought you were removing the line from setUpAllDevices method. I guess it's too late... :)